packages/volto-separator-block/src/components/Data.jsx

```diff
@@ -18,9 +21,12 @@ const SeparatorData = (props) => {
           [id]: value,
         });
       }}
+      onChangeBlock={onChangeBlock}
       formData={data}
       fieldIndex={data.index}
       block={block}
+      navRoot={navRoot}
+      contentType={contentType}
```

Comment:
For any props that aren't used directly here, you can do this on line 7:
```
const { data, block, schemaEnhancer, ...rest } = props;
```

And then pass them on together with `{...rest}` in the <BlockDataForm> component.

Reply:
I implemented that.